test/jdk/java/lang/invoke/indify/Indify.java

```diff
@@ -1228,6 +1244,55 @@ private static void writeOutputs(DataOutputStream out, Object... data) throws IO
         for (Object x : data)  writeOutput(out, x);
     }
 
+    public class Bytecode {
```

Comment:
You don't really need this class; you should be able to get whatever you need from ClassModel.

Reply:
I agree, this is just a temporary thing and it will be removed after, just to put everything together first and determine what exactly I will need from the classModel for the tool and then it will all be moved to the Indify class